src/ui-test/camelk_didact_tutorial_test.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+import {
+	ActivityBar,
+	By,
+	EditorView,
+	until,
+	ViewItem,
+	ViewSection,
+	WebView
+} from 'vscode-extension-tester';
+
+describe('Didact Tutorial of Camel K', function () {
+
+	let switchedContextWebview: WebView | undefined;
+	let handle: string | undefined;
+
+	before(async function () {
+		this.timeout(30000);
+	});
+
+	after(async function () {
+		if (switchedContextWebview) {
+			await switchBackWithAWorkaround();
+			switchedContextWebview = undefined;
+		}
+		await new EditorView().closeAllEditors();
+	});
+
+	it('Check that Camel K Didact tutorial opens with some content', async function () {
+		this.timeout(30000);
+		await startTutorial();
+
+		const didactTutorialWebView = new WebView();
+		const expectedTitle = 'Apache Camel K';
+		expect(await didactTutorialWebView.getTitle()).equal(expectedTitle);
+
+		await checkContentOfTutorial(didactTutorialWebView, expectedTitle);
+	});
+
+	async function checkContentOfTutorial(didactTutorialWebView: WebView, expectedTitle: string) {
+		await switchToFrameWithAWorkaround(didactTutorialWebView);
+		switchedContextWebview = didactTutorialWebView;
+		const titleElement = await didactTutorialWebView.findWebElement(By.css('h1'));
+		expect(await titleElement.getText()).equal(expectedTitle);
+	}
+
+	/**
+	 * See https://github.com/redhat-developer/vscode-extension-tester/issues/301
+	 */
+	async function switchToFrameWithAWorkaround(didactTutorialWebView: WebView) {
+		handle = await didactTutorialWebView.getDriver().getWindowHandle();
+		await didactTutorialWebView.getDriver().wait(until.elementLocated(By.className('webview ready')), 10000);
+		const frame = await didactTutorialWebView.getDriver().findElement(By.className('webview ready'));
+		await didactTutorialWebView.getDriver().switchTo().frame(frame);
+
+		await didactTutorialWebView.getDriver().wait(until.elementLocated(By.id('active-frame')), 10000);
+		const activeFrame = await didactTutorialWebView.getDriver().findElement(By.id('active-frame'));
+		await didactTutorialWebView.getDriver().switchTo().frame(activeFrame);
```

Comment:
I think it is better to keep i as close as possible to upstream instead of extracting a method with teh Locator in parameter https://github.com/redhat-developer/vscode-extension-tester/blob/a7fc9307658bebf2b7542331e422d8eb70568cd7/page-objects/src/components/editor/WebView.ts#L47-L53
The fix is already in master and so will eb available on next release of vscode-extension-tester